app/assets/stylesheets/components/_password_policy.scss

```diff
@@ -0,0 +1,12 @@
+.password-policy__label {
+  font-weight: bold;
+  margin-bottom: $baseline-unit;
+}
+
+.password-policy__list {
+  margin-bottom: 0.75rem;
```

Comment:
Vertical spacing should be done with reference to `$baseline-unit`. I believe `0.75rem` is `$baseline-unit * 2`.